chatmaild/src/chatmaild/tests/plugin.py

```diff
@@ -71,11 +71,11 @@ def maildata(request):
 
     assert datadir.exists(), datadir
 
-    def maildata(name, from_addr, to_addr):
+    def maildata(name, **kwargs):
```

Comment:
i suggest to be more conservative here: 
```suggestion
    def maildata(name, from_addr, to_addr,  **kwargs):
```